source/changelog.txt

```diff
@@ -24,6 +24,12 @@ v2.0.0
   - ``db.enableFreeMonitoring``
   - ``db.disableFreeMonitoring``
 
+v1.10.5
+-------
+*Released August 11, 2023*
+
+Provides an accessible list of ``mongosh`` `downloads <https://downloads.mongodb.com/compass/mongosh.json>`__.
```

Comment:
[suggestion]

I'm wondering if it would also make sense to mention this list of downloads on the Install page, maybe in a Learn More section at the bottom of the page. I'm a little concerned that if this is the only place we mention this list of downloads, we'll start needing to link to this particular section of the release notes, which can be confusing. This was a similar issue we had with the SBE docs before we created a specific page for SBE.

I think adding the Learn More section will also allow us to add a bit more context to this list, with a brief note about why you might want to access this download list. It could be something as simple as: "Use this list to programatically access mongosh binaries with your application." or something to that effect.

Reply:
Done, thank you!